From 720d9a1d31e400bddb4f3bcf0551f5a29f67d489 Mon Sep 17 00:00:00 2001
From: Avinash M N <avimn@amd.com>
Date: Thu, 22 Jun 2017 10:58:13 +0530
Subject: [PATCH 2057/2094] drm/amdgpu: Revert "remove the clearance of vce 4.0
 interrupt mask"

This reverts commit 7145ead3692d2f3cd4a554c266bc9a762d0868f8.
This commit is not present in amd-staging-4.11

Signed-off-by: Avinash M N <avimn@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/vce_v4_0.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/gpu/drm/amd/amdgpu/vce_v4_0.c b/drivers/gpu/drm/amd/amdgpu/vce_v4_0.c
index 07b2ac7..0012835 100644
--- a/drivers/gpu/drm/amd/amdgpu/vce_v4_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/vce_v4_0.c
@@ -977,6 +977,10 @@ static int vce_v4_0_process_interrupt(struct amdgpu_device *adev,
 {
 	DRM_DEBUG("IH: VCE\n");
 
+	WREG32_P(SOC15_REG_OFFSET(VCE, 0, mmVCE_SYS_INT_STATUS),
+			VCE_SYS_INT_STATUS__VCE_SYS_INT_TRAP_INTERRUPT_INT_MASK,
+			~VCE_SYS_INT_STATUS__VCE_SYS_INT_TRAP_INTERRUPT_INT_MASK);
+
 	switch (entry->src_data[0]) {
 	case 0:
 	case 1:
-- 
2.7.4

